### PR TITLE
update footer copyright with funtion，get current year

### PR DESCRIPTION
**Pull request by loda13**
update 
components/Footer.tsx

from 2022 to 2023

### CODE REVIEW

**Comment by oeyoews** [comment]
> update components/Footer.tsx
> 
> from 2022 to 2023

Maybe use `new Date().getFullYear()` is better

**Comment by loda13** [comment]
> > update components/Footer.tsx
> > from 2022 to 2023
> 
> Maybe use `new Date().getFullYear()` is better

good idea，thx

**Comment by loda13** [comment]
Visit this URL to preview：
https://nextjs-notion-starter-h92t0mivw-loda.vercel.app/

File: components/Footer.tsx [hunk 1]
```diff
@@ -33,9 +33,11 @@ export const FooterImpl: React.FC = () => {
     setHasMounted(true)
   }, [])
 
+  const currentYear  = new Date().getFullYear();
```
Comment (transitive-bullshit) [inline components/Footer.tsx:36]: this is probably fine.. but in general, injecting any date-related text into server-rendered pages can cause issues with react hydration

Comment (loda13) [reply components/Footer.tsx:36]: ok，old way is better #476

**Comment by transitive-bullshit** [comment]
Closing in favor of https://github.com/transitive-bullshit/nextjs-notion-starter-kit/pull/479/

**Pull request closed without merging**